api/handler/v1beta1/appeal.go

```diff
@@ -195,6 +195,44 @@ func (s *GRPCServer) RevokeAppeal(ctx context.Context, req *guardianv1beta1.Revo
 	}, nil
 }
 
+func (s *GRPCServer) RevokeAppeals(ctx context.Context, req *guardianv1beta1.RevokeAppealsRequest) (*guardianv1beta1.RevokeAppealsResponse, error) {
+	filters := &domain.ListAppealsFilter{}
+	if req.GetAccountId() != nil {
+		filters.AccountID = req.GetAccountId()
+	}
+	filters.Statuses = []string{domain.AppealStatusActive}
+	if req.GetProviderTypes() != nil {
+		filters.ProviderTypes = req.GetProviderTypes()
+	}
```

Comment:
```suggestion
	filters.ProviderTypes = req.GetProviderTypes()
```

We can just directly call this without checking nil, getter function will have nil checking inside it

Reply:
done